tests/integration/test_services_audio_preprocessor_analyzer_organizer.py

```
@@ -0,0 +1,1167 @@
+        wf.setnchannels(1)
+        wf.setsampwidth(2)
+        wf.setframerate(sample_rate)
+        # silence frames
+        wf.writeframes(struct.pack(f"<{num_samples}h", *([0] * num_samples)))
+    return path
+
+
+def _make_audio_metadata(
+    file_path: Path,
+    duration: float = 240.0,
+    **kwargs: Any,
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)